api/main.py

```diff
@@ -87,10 +87,9 @@ async def get_node(node_id: str):
 
 @app.get('/nodes', response_model=List[Node])
 async def get_nodes(request: Request):
-    query_param_dict = dict(request.query_params)
-    if 'parent' in query_param_dict:
-        query_param_dict = Node.modify_parent(query_param_dict)
-    return await db.find_by_attributes(Node, query_param_dict)
+    query_params = dict(request.query_params)
+    translated_params = Node.translate_fields(query_params)
```

Comment:
To reduce the LoC, I would suggest,

translated_params = Node.translate_fields(dict(request.query_params))

Reply:
I split this into 2 to make it easier to read and keep the line short.  Sometimes a smaller number of lines actually makes the code harder to read or maintain.